services/src/main/java/org/keycloak/theme/DefaultThemeSelectorProvider.java

```diff
@@ -47,6 +49,9 @@ public String getThemeName(Theme.Type type) {
 
         if (name == null || name.isEmpty()) {
             name = Config.scope("theme").get("default", Version.NAME.toLowerCase());
+            if ((type == Theme.Type.ACCOUNT) && isAccount2Enabled) {
+                name = name.concat(".v2");
+            }
```

Comment:
> Again, I'm not sure this is the right place to hardcode this. What if we have more `v2` themes in the future? Cannot we have some property for this is `Theme.Type`, perhaps?

This is what Stian wanted instead of the way I coded it before (which did use a property).

Reply:
Ok. Even though I'm a bit concerned about that we have the "v2" logic spread at many places (and therefore it won't be quite straightforward to e.g. add v2 to other themes, or v3 to this theme), I'm ok with it since @stianst is.